### PR TITLE
xfce4-13.xfce4-xkb-plugin: init at 0.8.1

### PR DESCRIPTION
**Pull request by volth**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by xeji** [comment]
@GrahamcOfBorg eval

**Comment by xeji** [comment]
merge conflict

File: pkgs/desktops/xfce4-13/default.nix [hunk 1]
```diff
@@ -82,7 +82,7 @@ makeScope newScope (self: with self; {
 
   xfce4-volumed-pulse = callPackage ./xfce4-volumed-pulse { };
 
-  xfce4-whiskermenu-plugin = callPackage ./xfce4-whiskermenu-plugin { };
```
Comment (xeji) [inline pkgs/desktops/xfce4-13/default.nix]: This was just added in #44317. Why  remove it now?

Comment (volth) [reply pkgs/desktops/xfce4-13/default.nix]: it should not be removed :(
Actual PR is https://github.com/NixOS/nixpkgs/pull/43778, I am just trying to split to to small pieces to get some traction.
But it results to merge collisions :(

**Pull request merged**